lua/gitlab/state.lua

```diff
@@ -133,15 +133,16 @@ M.settings = {
   popup = {
     width = "40%",
     height = "60%",
+    position = "50%",
     border = "rounded",
     opacity = 1.0,
-    edit = nil,
     comment = nil,
+    edit = nil,
     note = nil,
-    help = nil,
-    pipeline = nil,
```

Comment:
Why are we removing these settings? 

Additionally, if it's necessary to remove them they need to be added to our `health.lua` file under the `removed_settings_fields` section so that users know this functionality was deprecated.

Reply:
My motivation was that the help and pipeline popups' configuration is limited - their height and width are calculated automatically and I thought it could be confusing to have the possibility to configure them but the configuration would not be applied.

But on second thoughts I think some users just still might want to configure opacity and border for these separately, so I'm adding the settings back in.